app-src/player-router/.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "4.1"
```

Comment:
Oh, yeah, this should be moved to the root of the project, and Travis needs to be updated to point to this project.